benchmarks/Bench.cs

```diff
@@ -0,0 +1,81 @@
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Benchmarks
```

Comment:
namespace should be `TerraFX.Interop.Mimalloc.Benchmarks` to match other projects

Reply:
Changed to `TerraFX.Interop.Benchmarks` to make it consistent